### PR TITLE
Feature/search

### PR DESCRIPTION
**Pull request by MrTheGood**

### CODE REVIEW

File: Project-Starlord/Controllers/AccountController.cs [hunk 1]
```diff
@@ -147,6 +147,21 @@ public async Task<ActionResult<UserModel>> Register(UserModel userModel)
             return userModel;
         }
 
+        [Authorize]
+        [HttpGet("{query}")]
```
Comment (CJVisser) [inline Project-Starlord/Controllers/AccountController.cs]: Why is it called query the variable? I think filterValue would be a better name or something like that, But we can discuss this together.

**Review by CJVisser — COMMENTED** [review]
Everything good only naming to discuss

File: Project-Starlord/Controllers/AccountController.cs (hunk 1 above)
Comment (MrTheGood) [reply Project-Starlord/Controllers/AccountController.cs]: we can even call it `input` or indeed `filterValue`

Comment (CJVisser) [reply Project-Starlord/Controllers/AccountController.cs]: Can you call it filterValue? After that the PR is all good.

Comment (MrTheGood) [reply Project-Starlord/Controllers/AccountController.cs]: done

**Pull request merged**